### ISSUE TITLE
Spyder black screen after launching: "Failed to create D3D shaders"

### ISSUE THREAD

**Issue by Juanouo** [spyder-ide/spyder#7836]
## Problem Description
When I open Spyder it launches, but the screen is all black. The buttons and everything are still there but you can't see them. (If I click on the top left the file menu pops up). Anaconda prompt says "Failed to create D3D shaders".
 The last uncommon thing I did was adding python and conda to the path, and the next day I had this issue, after Spyder crashed. I tried everything in the troubleshooting guide: updating, uninstalling and reinstalling, reinstalling Anaconda, opening qtconsole and ipython and none of those worked. 

### What steps reproduce the problem?

1. Launch Spyder
2. Wait
3. Black window instead of UI

### What is the expected output? What do you see instead?
UI should be visible, but it isn't

### Paste Traceback/Error Below (if applicable)

"Failed to create D3D shaders".

## Versions

* Spyder version:  3.2.8
* Python version:  3.6.5
* Qt version: 5.9.5
* PyQt version: 5.9.2
* Operating System name/version: Windows 8

### Dependencies

```
IPython >=4.0    :  6.4.0 (OK)
cython >=0.21    :  0.28.2 (OK)
jedi >=0.9.0     :  0.12.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
numpy >=1.7      :  1.14.3 (OK)
pandas >=0.13.1  :  0.23.0 (OK)
pycodestyle >=2.3:  2.4.0 (OK)
pyflakes >=0.6.0 :  1.6.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pylint >=0.25    :  1.8.4 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
rope >=0.9.4     :  0.10.7 (OK)
sphinx >=0.6.6   :  1.7.4 (OK)
sympy >=0.7.3    :  1.1.1 (OK)

```
![image](https://user-images.githubusercontent.com/37188403/45109574-55b32200-b116-11e8-9d74-008dd39933ea.png)

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.1). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by Juanouo** [comment]
I updated Spyder using conda, then restarted my PC, but it still says it's in version 3.2.8. After that, I tried again to update it and it says "All requested packages already installed".  Also got a Microsoft .NET Framework error. When trying through the Anaconda Navigator, it says "The following packages will be modified" but the list is blank and my only option is to Cancel. I'm currently running Windows 8, and I recently got to download the Windows 8.1 .iso ,  so I'll see if updating Windows first fixes the problem.

**Comment by Juanouo** [comment]
After installing Windows 8.1, everything is working nicely again. Sorry for the inconvenience

**Comment by berkovsk** [comment]
After standart installing using Anaconda  Spyder  doesn't launch.  Conda shows error 
![default](https://user-images.githubusercontent.com/43699420/46244969-87569a80-c3ef-11e8-84aa-ec5179beca21.png)
 I tried everything in the troubleshooting guide: updating, uninstalling and reinstalling, reinstalling Anaconda, opening qtconsole and ipython and none of those worked. I work in Windows 7.
Here is my conda info
![default](https://user-images.githubusercontent.com/43699420/46245019-53c84000-c3f0-11e8-9d25-11ce8f298cb3.png)
Moreover, I tried to edit start.py (as it is  described in some sites) and install PyOpenGL, unfortunately, Spyder is still doesn't launch and shows a black screen with error messages in Аnaconda prompt. It's interesting, that on the other computer with   Windows 7  the same downloading (the same day, the same method) was succesfull and Spyder works correctly. What can I do?
